### PR TITLE
docs(odin-html) - more text styles md file translation

### PR DESCRIPTION
**Pull request by Doguhannilt**
# Description
Turkish translation of "more-text-styles" md file

Note: I didn't change some of the headers in the md file because they are related to the specific tag names. If it's necessary, drop a message about it.

### Checklist

Turkish transation of "more-text-styles" md file

Note: I didn't change some of the headers in the md file because they are related to the specific tag names. If it's necessary, drop a message about it.

- [ x] discord username: `doguhan_i#0001`
- [ ] Closes #<issue-number>
- [ ] PR must be created for an issue from issues under "In progress" column from [our project board](https://github.com/orgs/kamp-us/projects/2/views/1).
- [ x] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules, and should include the app/package/service name in the title. For example, a title like "docs(@kampus-apps/pano): Add README.md" can be used.
- [ ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [ ] I ran `npx turbo run` at the root of the repository, and build was successful.
- [ ] I installed the npm packages using `npm install --save-exact <package>` so my package is pinned to a specific npm version. Leave empty if no package was installed. Leave empty if no package was installed with this PR.

### How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

### CODE REVIEW

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 1]
```diff
@@ -1,61 +1,62 @@
-### Introduction
+### Giriş
 
-You've already done a bit of text-manipulation in our Foundations lessons. This lesson will cover a few more useful CSS properties that can be used when working with text.
+Daha önce Foundations derslerimizde biraz metin manipülasyonu yaptınız. Bu ders, metinle çalışırken kullanılabilecek birkaç daha kullanışlı CSS özelliğini kapsayacak.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Daha önce Temel Bilgiler derslerimizde birkaç metin manipülasyonu yaptınız. Bu ders, metinle çalışırken kullanılabilecek birkaç kullanışlı CSS özelliğini daha kapsayacak.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 2]
```diff
@@ -1,61 +1,62 @@
...
-* You'll learn how to use custom fonts on your web projects.
-* You'll learn some more text-related CSS properties.
+* Web projelerinizde özel fontları nasıl kullanacağınızı öğreneceksiniz.
+* Metinle ilgili birkaç daha fazla CSS özelliğini öğreneceksiniz.
 
-### Fonts
+### Yazı Tipleri
 
-In our Foundations lesson, we covered changing the `font-family` of an element, but there is some nuance and detail that we left out at the time.
+Foundations dersimizde bir öğenin `font-family`'sini değiştirmeyi ele almıştık, ancak o zaman bazı incelik ve detayları atlamıştık.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Temel Bilgiler dersimizde bir öğenin `font-family`'sini değiştirmeyi ele almıştık, ancak o zaman bazı incelik ve detayları atlamıştık.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 3]
```diff
@@ -1,61 +1,62 @@
...
 
-#### Online font libraries
+#### Çevrim içi font kütüphaneleri
 
-One popular and easy method to get fonts that are _not_ installed on a user's computer is to use an online font library like [Google Fonts](https://fonts.google.com/), [Font Library](https://fontlibrary.org/) or the premium, but non-free [Adobe Fonts](https://fonts.adobe.com/).
+Kullanıcının bilgisayarında _yüklü olmayan_ fontlara ulaşmanın popüler ve kolay bir yolu, [Google Fonts](https://fonts.google.com/), [Font Library](https://fontlibrary.org/) veya premium ancak ücretli [Adobe Fonts](https://fonts.adobe.com/) gibi çevrimiçi font kütüphanelerini kullanmaktır.
 
-To use a font from one of these libraries, go to the website, select a font and then copy a snippet from the website to import that font from their server into your website. You'll be given either a `<link>` tag to put in your HTML like so....
+
+Bu kütüphanelerden birinden bir font kullanmak için, ilgili web sitesine gidin, bir font seçin ve ardından web sitelerinden bir parça kodu kopyalayarak, o fontu kendi web sitenize sunucularından içe aktarabilirsiniz. Genellikle, HTML'nizde şu şekilde kullanmak için bir `<link>` etiketi alırsınız...
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Bu kütüphanelerden birinden bir font kullanmak için, web sitesine gidin, bir font seçin ve ardından bu font'u web sitenin sunucularından kendi web sitenize aktarmak için web sitesinden bir parçacık(snippet) kopyalayın.  HTML'nize koymanız için size bir `<link>` etiketi verilecektir, örneğin....
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 4]
```diff
@@ -1,61 +1,62 @@
...
+"Ayrıca, web üzerinden indirdiğiniz bir fontu kullanmak da mümkündür. CSS dosyanızda, özel bir fontu `@font-face` kuralını kullanarak içe aktarıp tanımlarsınız ve ardından onu diğer font-family'ler gibi kullanabilirsiniz. Birden fazla font dosya formatı türü bulunmaktadır ve bunlar hakkında daha fazla bilgiye [fileinfo.com's page on Font File Formats](https://fileinfo.com/filetypes/font) sayfasında detaylı olarak ulaşabilirsiniz. Ancak lütfen bir font dosya formatı seçerken dikkatli olun, çünkü bazıları tarayıcılar tarafından evrensel olarak desteklenmemektedir. Tarayıcılar ve destekledikleri font formatları hakkında bilgi için [W3 Schools' page on CSS Web Fonts](https://www.w3schools.com/css/css3_fonts.asp) sayfasını inceleyebilirsiniz."
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Web üzerinden indirdiğiniz bir fontu kullanmak da mümkündür. CSS dosyanızda, özel bir fontu `@font-face` kuralını kullanarak içe aktarıp ve tanımlarsınız ve ardından onu diğer font-family'ler gibi kullanabilirsiniz. Birden fazla font dosya formatı türü bulunmaktadır ve bunlar hakkında daha fazla bilgiye [fileinfo.com'un Font Dosya Biçimleri](https://fileinfo.com/filetypes/font) sayfasında detaylı olarak ulaşabilirsiniz. Ancak lütfen bir font dosya formatı seçerken dikkatli olun, çünkü bazıları tarayıcılar tarafından evrensel olarak desteklenmemektedir. Tarayıcılar ve destekledikleri font formatları hakkında bilgi için [W3 Schools'un CSS Web Yazı Tipleri](https://www.w3schools.com/css/css3_fonts.asp) sayfasını inceleyebilirsiniz."
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 5]
```diff
@@ -68,27 +69,27 @@ h1 {
...
 ```
 
-This method _may_ be more reliable than relying on a third-party font API, but it is always wise to include a fallback.
+Bu yöntem, üçüncü taraf bir font API'ına güvenmekten daha güvenilir olabilir, ancak her zaman bir yedek eklemek akıllıcadır.
 
-### Text styles
+### Metin stilleri
 
-You learned the basics of manipulating fonts in our Foundations lessons, but there is quite a bit more that you can do with CSS when it comes to manipulating text styles. These rules are all relatively simple and self-explanatory. You can refer to the docs for any questions you might have.
+Foundations derslerimizde fontları manipüle etmenin temellerini öğrendiniz, ancak CSS ile metin stillerini manipüle etmek konusunda daha fazla şey yapabilirsiniz. Bu kurallar genellikle basit ve açıklayıcıdır. Herhangi bir sorunuz varsa belgelere başvurabilirsiniz.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Temel Bilgiler derslerimizde fontları manipüle etmenin temellerini öğrendiniz, ancak CSS ile metin stillerini manipüle etmek konusunda daha fazla şey yapabilirsiniz. Bu kurallar genellikle basit ve açıklayıcıdır. Herhangi bir sorunuz varsa dökümanlara bakabilirsiniz.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 6]
```diff
@@ -68,27 +69,27 @@ h1 {
...
 
-For example, if you want all your header text to be italic you should use `font-style` to accomplish this. If you want some text in the _middle of a sentence_ to appear italic in order to emphasize that text, it is appropriate to use an `em` element. The [MDN doc on the Emphasis Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) puts stress on our point above.
+Örneğin, tüm başlık metninizi italik yapmak istiyorsanız, bunu gerçekleştirmek için `font-style`'ı kullanmalısınız. Eğer bir cümle içindeki metnin _orta kısmının_ italik görünmesini istiyorsanız ve bu metni vurgulamak istiyorsanız, bir `em` öğesini kullanmak uygun olacaktır. [MDN doc on the Emphasis Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) belgesi yukarıdaki noktayı vurgular.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Örneğin, tüm başlık metninizi italik yapmak istiyorsanız, bunu gerçekleştirmek için `font-style`'ı kullanmalısınız. Eğer bir cümle içindeki metnin _orta kısmının_ italik görünmesini istiyorsanız ve bu metni vurgulamak istiyorsanız, bir `em` öğesini kullanmak uygun olacaktır. [Emphasis Element üzerine MDN dökümanı](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em) yukarıdaki noktayı vurgular.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 7]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
 ```
 
 #### letter-spacing
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
#### letter-spacing(harf aralığı)
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 8]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
...
-Letter spacing does what you would expect.... it changes the space between letters in a word. This can be useful for adjusting custom fonts that you feel have too much or too little space. It can also be aesthetically pleasing in some cases, like headers.
+Harf aralığı, beklendiği gibi davranır... bir kelimenin harfleri arasındaki boşluğu değiştirir. Bu, fazla veya az boşluğa sahip olduğunu düşündüğünüz özel fontları ayarlamak için kullanışlı olabilir. Aynı zamanda bazı durumlarda, özellikle başlıklarda, estetik olarak hoş görünebilir.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
letter-spacing, beklendiği gibi davranır... bir kelimenin harfleri arasındaki boşluğu değiştirir. Bu, fazla veya az boşluğa sahip olduğunu düşündüğünüz özel fontları ayarlamak için kullanışlı olabilir. Aynı zamanda bazı durumlarda, özellikle başlıklarda, estetik olarak hoş görünebilir.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 9]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
...
+  aracılığıyla <a href="https://codepen.io">CodePen</a> üzerinde paylaşıldı.
+</span>
+
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-Obviously, use this sparingly and with care. Do not make your site hard to read!
+Tabii ki, bunu dikkatlice ve ölçülü bir şekilde kullanın. Sitenizi okunması zor hale getirmeyin!
 
 #### line-height
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
#### line-height(satır yüksekliği)
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 10]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
...
-Line height adjusts the space between lines in wrapped text. Adding a little line-height can increase readability.
+Satır yüksekliği, sarılı metindeki satırlar arasındaki boşluğu ayarlar. Biraz satır yüksekliği eklemek, okunabilirliği artırabilir.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
line-height, sarılı metindeki satırlar arasındaki boşluğu ayarlar. Biraz satır yüksekliği eklemek, okunabilirliği artırabilir.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 11]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
...
+
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
 #### text-transform
-Text transform simply changes the case of the given text. You can use this, for example, to force your heading tags to be all uppercase, or to capitalize every word.
+Text transform basitçe verilen metnin büyük-küçük harf durumunu değiştirir. Bu örneğin başlık etiketlerinizi tamamen büyük harfe zorlamak veya her kelimenin ilk harfini büyük yapmak için kullanılabilir.
 
-Usage is simple, and can be seen in the clear example on these [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).
+Kullanımı basittir ve açık bir örnek için bu [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform) görülebilir.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Kullanımı basittir ve açık bir örnek için bu [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)'dan bakabilirsiniz.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 12]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
...
 
 #### text-shadow
-As you might expect, `text-shadow` adds a shadow around the text in the selected element. This one is best used sparingly, but can be used to great effect in headings or other presentational text.
+Bekleyebileceğiniz gibi, `text-shadow` seçilen öğedeki metnin etrafına bir gölge ekler. Bu özellik genellikle özenle kullanılır, ancak başlıklar veya diğer sunum metinlerinde büyük etki yaratmak için kullanılabilir.
 
-The examples on the [MDN reference page for text-shadow](https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) show how to use it.
+Bu özelliği nasıl kullanacağınızı gösteren örnekler [MDN reference page for text-shadow]https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) bulunmaktadır.
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Bu özelliği nasıl kullanacağınızı gösteren örnekler [MDN'in text-shadow için referans sayfasında]https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow) bulunmaktadır.
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 13]
```diff
@@ -97,41 +98,47 @@ We should use the `em` element if italics is required for emphasis.
...
 
 #### ellipsis
-This one isn't a single property, but it's a useful trick to keep in your toolbox. With the `text-overflow` property, you can truncate overflowing text with an ellipsis. Making an overflow happen, however, requires the use of a couple other properties because the default behavior of text simply printing outside its container isn't technically considered an `overflow` (that's confusing, we know.  Sorry.)
+Bu, tek bir özellik değil, ancak takım çantanızda bulundurmanız gereken kullanışlı bir hile. text-overflow özelliği ile taşan metni üç nokta ile kırpabilirsiniz. Ancak bir taşmanın gerçekleşmesi için, metnin varsayılan davranışının basitçe konteynerının dışına basılması, teknik olarak bir overflow (taşma) olarak kabul edilmez (bu karmaşık, biliyoruz, özür dileriz).
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Bu, tek bir özellik değil, ancak takım çantanızda bulundurmanız gereken kullanışlı bir hile. `text-overflow` özelliği ile taşan metni üç nokta ile kırpabilirsiniz. Ancak bir taşmanın gerçekleşmesi için, metnin varsayılan davranışının basitçe konteynerının dışına basılması, teknik olarak bir `overflow(taşma)` olarak kabul edilmez (bu karmaşık, biliyoruz, özür dileriz).
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 14]
```diff
@@ -141,19 +148,19 @@ The full snippet is:
 }
 ```
 
-You can see more detail and an example in [this CSS Tricks Article](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/).  (Be ready to go look that article up every time you want to use this.)
+Bu örnekte daha fazla detay görebilirsiniz [this CSS Tricks Article](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/). (Bu özelliği kullanmak istediğinizde her seferinde o makaleyi aramak için hazır olun.)
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md]:
```suggestion
Daha fazla ayrıntı ve bir örneği [bu CSS Tricks makalesinde](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/) görebilirsiniz. (Bu özelliği kullanmak istediğinizde her seferinde o makaleyi aramak için hazır olun.)
```

File: content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 15]
```diff
@@ -1,61 +1,61 @@
-### Introduction
+### Giriş
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_css_concepts/more_text_styles.md:5]:
```suggestion
---
title: 'Diğer metin stilleri'
---

### Giriş
```

**Review by gulech — APPROVED** [review]
@Doguhannilt Thank you for all your effort. LGTM 🚀

**Pull request merged**